lightwallet-client-lib/src/main/java/co/electriccoin/lightwallet/client/internal/LightWalletClientImpl.kt

```diff
@@ -260,9 +260,16 @@ internal class LightWalletClientImpl private constructor(
     companion object {
         fun new(
             channelFactory: ChannelFactory,
-            lightWalletEndpoint: LightWalletEndpoint
+            lightWalletEndpoint: LightWalletEndpoint,
+            singleRequestTimeout: Duration = 10.seconds,
+            streamingRequestTimeout: Duration = 90.seconds
```

Comment:
Just FYI, iOS default values are 30s and 100s

Reply:
Thanks, these values are not changed by the PR, they're just moved around the code to have them accessible. We can revisit the default values topic later.